client/recoil.lua

```diff
@@ -1,158 +1,164 @@
-	-- [-102973651] = 0.3,		--['weapon_hatchet']
-	-- [-656458692] = 0.3,		--['weapon_knuckle']
-	-- [-1716189206] = 0.3,		--['weapon_knife']
-	-- [-581044007] = 0.3,		--['weapon_machete']
-	-- [-538741184] = 0.3,		--['weapon_switchblade']
-	-- [1737195953] = 0.3,		--['weapon_nightstick']
-	-- [419712736] = 0.3,		--['weapon_wrench']
-	-- [-853065399] = 0.3,		--['weapon_battleaxe']
-	-- [-1810795771] = 0.3,		--['weapon_poolcue']
-	-- [-2000187721] = 0.0,		--['weapon_briefcase']
-	-- [28811031] = 0.0,		--['weapon_briefcase_02']
-	-- [-499989876] = 0.0,		--['weapon_garbagebag']
-	-- [-800287667] = 0.0,		--['weapon_handcuffs']
-	-- [1309015656] = 0.0,		--['weapon_bread']
-	-- [940833800] = 0.3,		--['weapon_stone_hatchet']
+    -- Melee
+    -- [-1834847097] = 0.3,        --['weapon_dagger']
+    -- [-1786099057] = 0.3,        --['weapon_bat']
+    -- [-102323637] = 0.3,        --['weapon_bottle']
+    -- [2067956739] = 0.3,        --['weapon_crowbar']
+    -- [-1951375401] = 0.3,        --['weapon_flashlight']
+    -- [1141786504] = 0.3,        --['weapon_golfclub']
+    -- [1317494643] = 0.3,        --['weapon_hammer']
+    -- [-102973651] = 0.3,        --['weapon_hatchet']
+    -- [-656458692] = 0.3,        --['weapon_knuckle']
+    -- [-1716189206] = 0.3,        --['weapon_knife']
+    -- [-581044007] = 0.3,        --['weapon_machete']
+    -- [-538741184] = 0.3,        --['weapon_switchblade']
+    -- [1737195953] = 0.3,        --['weapon_nightstick']
+    -- [419712736] = 0.3,        --['weapon_wrench']
+    -- [-853065399] = 0.3,        --['weapon_battleaxe']
+    -- [-1810795771] = 0.3,        --['weapon_poolcue']
+    -- [-2000187721] = 0.0,        --['weapon_briefcase']
+    -- [28811031] = 0.0,        --['weapon_briefcase_02']
+    -- [-499989876] = 0.0,        --['weapon_garbagebag']
+    -- [-800287667] = 0.0,        --['weapon_handcuffs']
+    -- [1309015656] = 0.0,        --['weapon_bread']
+    -- [940833800] = 0.3,        --['weapon_stone_hatchet']
 
-	-- Handguns
-	[453432689] = 0.3,			--['weapon_pistol']
-	[-1075685676] = 0.5,		--['weapon_pistol_mk2']
-	[1593441988] = 0.2,			--['weapon_combatpistol']
-	[584646201] = 0.3,			--['weapon_appistol']
-	[911657153] = 0.1,			--['weapon_stungun']
-	[-1716589765] = 0.6,		--['weapon_pistol50']
-	[-1076751822] = 0.2,		--['weapon_snspistol']
-	[-771403250] = 0.5,			--['weapon_heavypistol']
-	[137902532] = 0.4,			--['weapon_vintagepistol']
-	[1198879012] = 0.9,			--['weapon_flaregun']
-	[-598887786] = 0.9,			--['weapon_marksmanpistol']
-	[-1045183535] = 0.6,		--['weapon_revolver']
-	[-879347409] = 0.6,			--['weapon_revolver_mk2']
-	[-1746263880] = 0.3,		--['weapon_doubleaction']
-	[-2009644972] = 0.3,		--['weapon_snspistol_mk2']
-	[-1355376991] = 0.3,		--['weapon_raypistol']
-	[727643628] = 0.3,			--['weapon_ceramicpistol']
-	[-1853920116] = 0.3,		--['weapon_navyrevolver']
-	[1470379660] = 0.3,			--['weapon_gadgetpistol']
+    -- Handguns
+    [453432689] = 13.3,            --['weapon_pistol']
```

Comment:
Why the large change in value? Is this left over from testing?

Reply:
Oh yes my mistake i was doing diagnostics to see if it actually in fact works seems like it only mostly effects fpp. Ill change that back tonight and fix those locales. First time doing one of those files.